### ISSUE TITLE
Test fail locally MLNMapAccessibilityElementTests testPlaceFeatureValues

### ISSUE THREAD

**Issue by louwers** [maplibre/maplibre-native#1973]
> ((element.accessibilityValue) equal to (@"31,337 feet")) failed: ("1.337 metres") is not equal to ("31,337 feet")

https://github.com/maplibre/maplibre-native/blob/8820165ccafb4f3db109b0b4ae91739d577d7ece/platform/ios/test/MLNMapAccessibilityElementTests.m#L52

This test fails locally.
